### ISSUE TITLE
Incompatible With Optifine 1.12

### ISSUE THREAD

**Issue by ElectricBlue1** [lumien231/Custom-Main-Menu#195]
I have an issue with "OptiFine 1.12 HD U C2" where the text/buttons and gui aren't showing
**Regular:** 
![capture1](https://user-images.githubusercontent.com/14302401/27261333-07f3c142-540f-11e7-87a2-6e398e995178.PNG)
**Optifine:**
![capture2](https://user-images.githubusercontent.com/14302401/27261342-1f6a2104-540f-11e7-8a26-90f032faf612.PNG)
![capture3](https://user-images.githubusercontent.com/14302401/27261343-1f70c7d4-540f-11e7-9f32-5067e6ad1420.PNG)

**Comment by lumien231** [comment]
Could you post your json file / resources? Can't reproduce this with the default menu. Also is the third screenshot a custom gui by you or a vanilla gui?

**Comment by ElectricBlue1** [comment]
Yes the third screenshot is the vanilla gui. Here is my resources and config
[CMM.zip](https://github.com/lumien231/Custom-Main-Menu/files/1085472/CMM.zip)

**Comment by lumien231** [comment]
If the third screenshot is a vanilla gui and it's still not working maybe the issue is between optifine and vanilla? Do Vanilla GUIs work if CMM isn't installed?

**Comment by ElectricBlue1** [comment]
No, it's Emojiful.
